### PR TITLE
:tada: Major Update: Add C1 support to C0VM.ts

### PR DESCRIPTION
**Pull request by MarkChenYutian**
# Current Progress

## Support `void*`  (Tagged Pointer) in C1 Language Standard

**Tabular Debug Mode**
image

**Graph Debug Mode**
image

## Support `invokeDynamic` (Function Pointer) in C1 Language Standard

** Tabular Debug Mode**
image

**Graph Debug Mode**
image

### CODE REVIEW

**Comment by MarkChenYutian** [comment]
@chaosarium The import folder feature is broken n commit #220bc74 , can you take a look on that when you have time?

**Comment by chaosarium** [comment]
Should have been fixed. It's just a missing argument somewhere in the long chain of passing `handle_import_folder` from top to bottom. Feel free to get rid of the `console.debug` I've added, but I feel like keeping some verbose mode may help future development. 

Also, I'm not sure why `LoadDocumentPlugin` takes `update_title` as an optional argument. But I followed the example and did the same with `handle_import_folder`.

File: src/components/code_editor/c0-editor.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ export default class C0Editor extends React.Component<C0EditorProps>
                         value = {this.props.editorValue}
                         extensions={[
                             breakpoint_extension,
-                            LoadDocumentPlugin(".c0", this.props.updateName, this.props.handle_import_folder),
+                            LoadDocumentPlugin(".c0, .c1", this.props.updateName, this.props.handle_import_folder),
```
Comment (chaosarium) [inline src/components/code_editor/c0-editor.tsx:45]: Make sure to pass `handle_import_folder`

**Comment by MarkChenYutian** [comment]
I've put all `console.debug` into `if (globalThis.DEBUG) {...}` so that everything is clean when running in normal mode.

**Pull request merged**